### DISCUSSION TITLE
Different rates based on location such as State or Zip Code

### DISCUSSION THREAD

**Discussion by rgouldkc** [CMSgov/price-transparency-guide#189] [Q&A] [answered]
We are a regional PPO network in the Kansas City metro. We have contacts with providers that work on both sides of the KS/MO state line that are reimbursed differently based on where the service is performed. Some will distinguish base on the state, Others will distinguish based on Zip Code. Unfortunately I do not see any way in the schema to indicate this, short of moving to my own derived schema for the in network rates file.

Anyone have any ideas?

**Comment by jhstafford** [comment]
Sorry I don't, but we have similar situations with some of our clients. The format is too 'provider-centric' for the FFS model. In some cases, the provider ID is irrelevant to the pricing, as long as they are in the network. Or as in your situation, there are a small number of area-based groupings. I guess we could create a file for each provider, with a complete CPT fee schedule for each one.

Sounds like a  solid governmental approach.

**Reply by RLTx1391** [reply]
Has anyone figured out a way to do zip codes?

**Reply by rgouldkc** [reply]
Not yet that I'm aware of. States are another similar problem when you are in a bi-state metro area - Kansas City for example.

**Comment by rakeshkatte** [comment]
We are planning to group the provider records based on zip or state and
associate the pricing to these groups separately.

**Reply by rgouldkc** [reply]
Basically I think that may work. The question I have is since there are probably in our case 2 rates based which state or what zip code(s) the service is performed in there would need to be something to indicate that and associated with the rate. My first thought is possibly to use the additional_information field to indicate that. I'm going to try that approach sometime this week and see what the result looks like. Thanks for the idea.

**Comment by shaselton-usds** [comment, marked as answer]
@rgouldkc @RLTx1391 I think the `additional_information` field could help out here in providing additional context to the negotiated rate. 

If we see rates dependent on zip codes to be a frequent occurrence in the contracts, potentially adding additional attributes to specifically capture this information could make sense.

**Reply by rgouldkc** [reply]
We do have a number of contracts where rates are zip code dependent. That being said, a recent termination with another regional PPO may have technically made those obsolete. The situation was providers that were in one of our networks and the other PPOs network would use the other networks rates in 2 counties, which we distinguished by zip codes. 14 of those contracts are still active and in use. Whether those 14 terminate soon or not remains to be seen.

And since we can currently accommodate different rates by zip code, state, address, place of service code and Peer Group (for work comp), as long as those contracts are in use we need to be able to accommodate them. And I can't say there would not be new contracts set up/created in the future that use any of these.

If we can at least use the additional_information field for our purposes I can solve my problem for now.
